Graylog2/graylog2-server
# Search filters are now stored in Content Packs.

## Description
Search filters are now stored in Content Packs. Implements #12719.

## Motivation and Context
Query and Search classes implement ContentPackable interface.
Saved/Named Search (a view) can be part of the Content Pack.

We have to ensure that when a Search contains queries with Search Filters, they are stored in Content Pack and installed during this Content Pack installation.

## How Has This Been Tested?
As long as FE is not yet ready with Search Filters, we have to do the following:
1. Create a Search with Search Filters through API (or by changing existing Search in MongoDB by adding Search Filters).
2. Save this Search as a View Mongo DB collection.
3. Create a new Content Pack in System->Content Packs part of UI (remember to select proper Search).
4. Verify that content_packs collection in Mongo DB stores a Search together with Search Filters.
5. Install a Content Pack.
6. Verify that after its installation a new Search is present and it has proper Search Filters.

## Types of changes


## Checklist:
